# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ppp/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ppp-ppp-2.4.9/pppd/Makefile.linux.vanilla	2023-05-15 13:12:49.728266416 +0200
+++ ppp-ppp-2.4.9/pppd/Makefile.linux	2023-05-15 13:14:06.707271974 +0200
@@ -155,7 +155,7 @@
 
 ifdef NEEDDES
 ifndef USE_CRYPT
-CFLAGS   += -I$(shell $(CC) --print-sysroot)/usr/include/openssl
+CFLAGS   += 
 NEEDCRYPTOLIB = y
 else
 CFLAGS   += -DUSE_CRYPT=1
